ompi/mpi/c/comm_accept.c

```diff
@@ -89,6 +92,10 @@ int MPI_Comm_accept(const char *port_name, MPI_Info info, int root,
         }
     }
 
+    if (!ompi_mpi_dynamics_is_enabled(FUNC_NAME)) {
+        return OMPI_ERRHANDLER_INVOKE(comm, MPI_ERR_SPAWN, FUNC_NAME);
+    }
+
     /* parse info object. no prefedined values for this function in MPI-2
```

Comment:
@rhc54 Yes, I mean "not supported".  I just double checked the code, and I used "not supported" in all relevant places (not "not implemented").

(found your line comment in the outdated diff on github :-) )